gwctl/pkg/printer/policies.go

```diff
@@ -148,3 +151,88 @@ func (pp *PoliciesPrinter) PrintDescribeView(policies []policymanager.Policy) {
 		}
 	}
 }
+
+type policyCrdDescribeView struct {
+	Name        string                 `json:",omitempty"`
+	Namespace   string                 `json:",omitempty"`
+	APIVersion  string                 `json:",omitempty"`
+	Kind        string                 `json:",omitempty"`
+	Metadata    map[string]interface{} `json:",omitempty"`
+	Spec        map[string]interface{} `json:",omitempty"`
+	Status      map[string]interface{} `json:",omitempty"`
+}
+
+func (pp *PoliciesPrinter) PolicyCrd_PrintDescribeView(policyCrds []policymanager.PolicyCRD) {
```

Comment:
This isn't a blocker, but it would be nice if we could adjust the overall evolving names of other functions as well:

```
PrintPoliciesGetView
PrintPoliciesDescribeView
PrintPolicyCRDsGetView
PrintPolicyCRDsDescribeView
```

If we do include this change, lets correspondingly change the Test function names as well :)